pkgs/tools/audio/beets/default.nix

```diff
@@ -1,24 +1,23 @@
 { stdenv, lib, fetchFromGitHub, writeScript, glibcLocales, diffPlugins, substituteAll
 , pythonPackages, imagemagick, gobject-introspection, gst_all_1
-, runtimeShell
-, unstableGitUpdater
+, runtimeShell, unstableGitUpdater
```

Comment:
If this commit is only a reformat, I'm up to putting each input in it's own line.

Reply:
I would format it with nixpkgs-fmt which would remove lots of the formatting.